src/vcpkg-test/system.cpp

```diff
@@ -5,9 +5,11 @@
 #include <vcpkg/base/optional.h>
 #include <vcpkg/base/strings.h>
 #include <vcpkg/base/stringview.h>
+#include <vcpkg/base/system.debug.h>
 #include <vcpkg/base/system.h>
 #include <vcpkg/base/system.process.h>
 
+#include <iostream>
```

Comment:
Do we use iostream for anything else? I thought we burninated all use of it.

Reply:
No, I used this for debugging